src/services/storage.js

```diff
@@ -1,9 +1,33 @@
 import {createState} from '@hookstate/core'
 
-const starCount = createState(5);
+const starCount = createState(0);
+const currentIndex = createState(null);
+const favouriteMovies = createState([]);
 
-const increaseStarCount = () => {
-    starCount.set(p => p + 1);
+const addFavourite = (movie) => {
+    favouriteMovies.set([...favouriteMovies, movie]);
+    console.log(favouriteMovies.get());
+};
+
+const removeFavourite = (movie) => {
+    const filteredList = favouriteMovies.filter(
+        item => item !== movie
+      );
+      favouriteMovies.set(filteredList);
+}
```

Comment:
After fixing adding favourite movies, I will continue to work on the remove movies from the array